scripts/cspell/project-words.txt

```diff
@@ -36,8 +36,8 @@ hostnames
 hostport
 htpasswd
 ingester
-ingester's
 ingesters
+ingester's
```

Comment:
Please revert order change

Reply:
When the change is reverted and the make spellcheck command is run, the output displays a sorting error -

cat scripts/cspell/project-names.txt | grep -v '^#' | grep -v '^\s*$' | tr ' ' '\n' > scripts/cspell/project-names-parsed.txt
cd scripts/cspell && ./spellcheck.sh
sort: project-words.txt:40: disorder: ingesters
project-words.txt is not sorted.
make: *** [Makefile:60: spellcheck] Error 1